SECURITY.md

```diff
@@ -0,0 +1,3 @@
+# Security Policy
+
+If you have identified a vulnerability in this repository, please file [an issue using our report template](https://github.com/thisdot/starter.dev-github-showcases/issues/new?assignees=&labels=security&template=security_report.yml&title=%5BSECURITY%5D%3A+). Our team will review the issue and determine next steps.
```

Comment:
Is there meant to be an assignee value in this URL? Or just the key?

Reply:
This is the default that GitHub generates so I went with it. There's not really a reason for it. 🤷